identity/app/clients/DiscussionClient.scala

```diff
@@ -82,4 +94,29 @@ class DiscussionClient(wsClient: WSClient, config: conf.IdentityConfiguration)(i
       case e => Future.failed(new DiscussionApiServiceException(s"findProfileStats failure to find $userId", e))
     }
   }
+
+  /**
+    * Check if the user has at least one comment. This is a lightweight check which replaces
+    * the findProfileStats call, which has been found to be extremely slow for users with a large
+    * number of comments, as on the Discussion API side it involves a full count of all comments.
+    * This method simply requests a single comment, and checks if the response is non-empty.
+    * If it is, the user has at least one publicly visible comment, and their profile can be shown.
+    */
```

Comment:
As this is a simpler, quicker solution, no need to mention the slower and more complex solution.  Unless there's something about the more complex solution that's going to tempt people to reinstate it.
```suggestion
  /**
    * Check if the user has at least one comment.
    * This method simply requests a single comment, and checks if the response is non-empty.
    * If it is, the user has at least one publicly visible comment, and their profile can be shown.
    */
```